jobdispatcher/src/test/java/com/firebase/jobdispatcher/DefaultJobValidatorTest.java

```diff
@@ -56,25 +58,25 @@ public void testValidate_retryStrategy() throws Exception {
         Map<RetryStrategy, List<String>> testCases = new HashMap<>();
         testCases.put(
             new RetryStrategy(0 /* bad policy */, 30, 3600),
-            Arrays.asList("Unknown retry policy provided"));
+            Collections.singletonList("Unknown retry policy provided"));
         testCases.put(
             new RetryStrategy(RetryStrategy.RETRY_POLICY_LINEAR, 15, 3600),
-            Arrays.asList("Initial backoff must be at least 30s"));
+            Collections.singletonList("Initial backoff must be at least 30s"));
         testCases.put(
             new RetryStrategy(RetryStrategy.RETRY_POLICY_EXPONENTIAL, 15, 3600),
-            Arrays.asList("Initial backoff must be at least 30s"));
+            Collections.singletonList("Initial backoff must be at least 30s"));
         testCases.put(
             new RetryStrategy(RetryStrategy.RETRY_POLICY_LINEAR, 30, 60),
-            Arrays.asList("Maximum backoff must be greater than 300s (5 minutes)"));
+            Collections.singletonList("Maximum backoff must be greater than 300s (5 minutes)"));
         testCases.put(
             new RetryStrategy(RetryStrategy.RETRY_POLICY_EXPONENTIAL, 30, 60),
-            Arrays.asList("Maximum backoff must be greater than 300s (5 minutes)"));
+            Collections.singletonList("Maximum backoff must be greater than 300s (5 minutes)"));
         testCases.put(
             new RetryStrategy(RetryStrategy.RETRY_POLICY_LINEAR, 301, 300),
-            Arrays.asList("Maximum backoff must be greater than or equal to initial backoff"));
+            Collections.singletonList("Maximum backoff must be greater than or equal to initial backoff"));
         testCases.put(
             new RetryStrategy(RetryStrategy.RETRY_POLICY_EXPONENTIAL, 301, 300),
-            Arrays.asList("Maximum backoff must be greater than or equal to initial backoff"));
+            Collections.singletonList("Maximum backoff must be greater than or equal to initial backoff"));
```

Comment:
These lines are too long (> 100) with the full method name. Can you static import Collections.singletonList?